KarhooUISDK/Screens/PrebookConfirmationScreen/KarhooBookingConfirmationViewModel.swift

```diff
@@ -132,6 +140,16 @@ class KarhooBookingConfirmationViewModel: BookingConfirmationViewModel {
         callback()
     }
     
+    func onAddToCalendar(viewModel: KarhooAddToCalendarView.ViewModel) {
```

Comment:
I'd change the method name to something like `bind actions` of add-to-calendar view model